CONTRIBUTING.md

```diff
@@ -1,53 +1,307 @@
+
+### Issue management
+
+We have some issue automation to close inactive issues and create/pin/archive the weekly meeting issues.
+More information is in the [Appendix: Issue automation](#appendix-issue-automation) section.
+
+Everyone is encouraged to open and comment on issues in the project.
+If you want to tag/assign/close something and you don't have enough permissions, add a comment and someone will help.
+
+Issues are managed by the [Triage](#triage), [Maintainer](#maintainer) and [TSC](#tsc) teams.
+They may move issues to other repositories within the project as needed.
+
+In order to keep the issues list manageable and realistic for a relatively small group of volunteers, issues are proactively closed when it's not clear that they can be completed.
+Issues may be closed when:
+
+- they have been inactive for a long time
+- they are out of scope or no further constructive action can be taken
+- they are complete (yay!)
+- they are unclear and more details are not forthcoming
+- as a group, there is agreement that no further action will be taken
+
+When issues are closed, a comment is added about why.
+Closing issues is a reversible action, and it is always acceptable to comment and explain (politely) why an issue should not have been closed.
+
+### Labels
+
+We make extensive use of labels.
+The main categories are:
 
-### Using forks
+- [Housekeeping](https://github.com/OAI/OpenAPI-Specification/labels/Housekeeping) for meetings, project logistics, etc.
+- [approved pr port](https://github.com/OAI/OpenAPI-Specification/labels/approved%20pr%20port) for pull requests that repeat a change from one version to another
+- most other tags are used to group similar or related issues into topic areas; this list is ever-changing
 
-All work **MUST be done on a fork**, using a branch from the _earliest relevant and [active](#active-branches)_ `vX.Y-dev` branch, and then submitted as a PR to that `vX.Y-dev` branch.
-For example, if a change in November 2024 apples to both 3.1 and 3.2, the PR would go to the `v3.1-dev` branch, which will be merged up to `v3.2-dev` before the next 3.2 release.
+Labels related to [issue automation](#appendix-issue-automation)
+
+- [Needs attention](https://github.com/OAI/OpenAPI-Specification/labels/Needs%20attention) automated tag when an issue is updated
+- [Needs author feedback](https://github.com/OAI/OpenAPI-Specification/labels/Needs%20author%20feedback) used to indicate that more information is needed from the issue creator
+- [No recent activity](https://github.com/OAI/OpenAPI-Specification/labels/No%20recent%20activity) if no information is received, the issue is marked for closure (automatic after 30 days)
+
+### Milestones
+
+We use milestones in GitHub to plan what should be included in future releases.
+Issues and pull requests should both be added to the milestone we expect they will be released in.
```

Comment:
I made a point to put PRs in milestones, and then when we released 3.0.4 and 3.1.1 I realized that you can look them up by target branch, which for us corresponds directly to releases.  So maybe we don't need to put milestones on PRs after all?  They should definitely go on issues, though.

Reply:
I think we can leave this instruction as-is for now, it's not a problem if we do milestone the PRs but it's a good tip about the branches! And it can be a nice visual indicators on the PRs as well if do end up holding them open for a while.